packages/trpc/server/routers/viewer/eventTypes/getByViewer.handler.ts

```diff
@@ -1,20 +1,16 @@
+import { Prisma } from "@prisma/client";
 // eslint-disable-next-line no-restricted-imports
```

Comment:
Reverted this file to -> v3.7.11.

<img width="1090" alt="Screenshot 2024-02-07 at 11 02 09 PM" src="https://github.com/calcom/cal.com/assets/1780212/31122c20-6291-4148-a451-5ac9c119756b">